modules/oe_theme_content_organisation/oe_theme_content_organisation.module

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Organisation content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\media\MediaInterface;
+use Drupal\media_avportal\Plugin\media\Source\MediaAvPortalPhotoSource;
+use Drupal\media\Plugin\media\Source\Image;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function oe_theme_content_organisation_preprocess_oe_theme_helper_in_page_navigation(array &$variables): void {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['entity'];
+
+  if ($node->get('oe_organisation_logo')->isEmpty()) {
+    return;
+  }
+
+  /** @var \Drupal\media\Entity\Media $media */
+  $media = $node->get('oe_organisation_logo')->entity;
+
+  if (!$media instanceof MediaInterface) {
+    // The media entity is not available anymore, bail out.
+    return;
+  }
+
+  // @TODO: Replace below with calling view() on the logo field
+  // once FeaturedMediaFormatter is avail.
+  $source = $media->getSource();
+
+  // If its an image media, render it and assign it to the image variable.
+  if (!$source instanceof MediaAvPortalPhotoSource && !$source instanceof Image) {
+   return;
+  }
+
+  $thumbnail = $media->get('thumbnail')->first();
+  $variables['image'] = ImageValueObject::fromStyledImageItem($thumbnail, 'oe_theme_ratio_3_2_medium');
+
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+  $cacheability->addCacheableDependency($media);
```

Comment:
I would also suggest to add file entity to the cache dependency because file can be changed externally.

Reply:
I've tested this, does not seem to be needed, lets discuss